src/actions/UserActions.ts

```diff
@@ -1,16 +1,27 @@
 import { Dispatch } from "redux";
-import { USER_FAIL, USER_LOADING, USER_SUCCESS, UserDispatchTypes } from "./UserActionTypes";
+import { USER_FAIL, USER_LOADING, USER_SUCCESS, UserDispatchTypes, UserType } from "./UserActionTypes";
 import axios from "axios";
 
+interface ResponseDataType {
+  data: UserType[],
+  page: number,
+  per_page: number,
+  total: number,
+  total_pages: number,
+  support: {
+    text: string
+    url: string
+  }
+}
+
 export const GetUser = (pn: number) => {
```

Comment:
React functional components should start with Uppercase.
Rest javascript functions including action creators should start with a lowercase letter.